backend/docker-compose.test-backend.yaml

```diff
@@ -1,30 +1,5 @@
 version: '3'
 services:
-    testing:
```

Comment:
what env variables are passed to the testing container if these are taken out?

Reply:
we pass the environment when running docker run (the testing container is run seperately now so we can copy the coverage file)